python/pyarrow/tests/test_csv.py

```diff
@@ -1507,8 +1488,8 @@ def test_stress_block_sizes(self):
 class TestSerialStreamingCSVRead(BaseTestStreamingCSVRead, unittest.TestCase):
 
     def open_csv(self, *args, **kwargs):
-        read_options = kwargs.setdefault('read_options', ReadOptions())
-        read_options.use_threads = False
+        # read_options = kwargs.setdefault('read_options', ReadOptions())
+        # read_options.use_threads = False
```

Comment:
Is this still to test SerialStreamingCSV? Should there be two classes so that all test get run for serial and non serial?

Reply:
Ah, good point (and clumsy of me to leave those comments in there).  I've changed up the test so now there is no base class and every test is parameterized on use_threads=True/False.